.rtd-environment.yml

```diff
@@ -6,13 +6,13 @@ channels:
 dependencies:
   - python=3.6
   - IPython=6.1.0
-  - astropy=3.0
+  - astropy=3
```

Comment:
Everything else is very much pinned to one version. Is that OK to have astropy's less constraint? (I think probably it's fine as we don't break anything in bugfixes, right?)

Reply:
That was my thinking, but I didn't explain that well!